dateparser_scripts/order_languages.py

```diff
@@ -9,7 +9,15 @@
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 
 # Languages with insufficient translation data are excluded
-avoid_languages = ['cu', 'kkj', 'nds', 'prg', 'tk', 'vai', 'vai-Latn', 'vai-Vaii', 'vo']
+avoid_languages = {'cu', 'kkj', 'nds', 'prg', 'tk', 'vai', 'vai-Latn', 'vai-Vaii', 'vo'}
+
+# Order from https://w3techs.com/technologies/overview/content_language
+# Last updated on 30.10.2020
+most_common_locales = [
+    'en', 'ru', 'es', 'tr', 'fa', 'fr', 'de', 'ja', 'pt', 'vi', 'zh', 'ar', 'it', 'pl', 'in', 'el',
+    'nl', 'ko', 'th', 'iw', 'uk', 'cs', 'sv', 'ro', 'hu', 'da', 'sr', 'sk', 'fi', 'bg', 'hr', 'lt',
+    'hi', 'no_NO', 'sl', 'no', 'et', 'lv'
+]
```

Comment:
Hi @mirceachira, I rechecked this list before merging and it seems that "Hebrew", "Norwegian Bokmål" and "Norwegian (Nynorsk)" were wrong (those codes you wrote are used in other specifications, but they are not in the `language_order` list). Could you change it? Thanks!

```suggestion
most_common_locales = [
    'en', 'ru', 'es', 'tr', 'fa', 'fr', 'de', 'ja', 'pt', 'vi', 'zh', 'ar', 'it', 'pl', 'in', 'el',
    'nl', 'ko', 'th', 'he', 'uk', 'cs', 'sv', 'ro', 'hu', 'da', 'sr', 'sk', 'fi', 'bg', 'hr', 'lt',
    'hi', 'nb', 'sl', 'nn', 'et', 'lv'
]
```

Reply:
(I have just seen that you mentioned them as "new" in this comment: https://github.com/scrapinghub/dateparser/issues/714#issuecomment-701373790, but I didn't see that before, sorry)